src/test/common.go

```diff
@@ -166,9 +166,9 @@ func (e2e *UDSE2ETest) DownloadZarfInitPkg(t *testing.T, zarfVersion string) {
 }
 
 // CreateZarfPkg creates a Zarf package in the given path
-func (e2e *UDSE2ETest) CreateZarfPkg(t *testing.T, path string, forceCreate bool) {
+func (e2e *UDSE2ETest) CreateZarfPkg(t *testing.T, path string, forceCreate bool, arch string) {
```

Comment:
i originally create a duplicate function with the arch param, but then switched this one to take the param. thoughts?

Reply:
ended up doing the duplicate func after all to pass the arch as a cli flag to the create command.